script/Ciel/Day 12/12_01_CIEL12_2B.txt

```diff
@@ -367,12 +389,14 @@ You're always the one doing the visiting, so I figured I'd head over to you for
 [sha:0b43d753990ce49514161134cd63025b23561051]{
 -- Page 55, Offset 31004.
 -- なにか不名誉な指摘を受けている気がしますがっ」
-It feels like you're pointing out something shameful."
+I feel like I'm quietly being judged for something."
 }
 [sha:090e5a5170d1a2bb722219aef2c65a51f49cdc3e]{
 -- Page 56, Offset 31005.
 -- 「いや、茶道部の部費でお茶飲んでるだけじゃなくて良かったなと。うん、感心感心。分かってましたけど、真面目に授業を受けてたんですね」
-"No, I'm glad you're not just wasting the tea ceremony club's budget and drinking tea all the time. Yes, how very admirable. I already knew this beforehand, but you really are taking school seriously."
+"No, I'm simply glad to hear you don't just sit around and drink tea on the tea ceremony club's dime.
+ Yeah, I've got to hand it to you.
+ You're really taking school seriously. Not that I had any doubts, of course."
```

Comment:
You're really > You really are

She said at some point earlier that she actually does take school seriously when they're talking (a lie, but nevertheless), so this is him affirming her statement.